clusterloader2/pkg/measurement/util/object_store.go

```diff
@@ -34,56 +35,73 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
-// PodStore is a convenient wrapper around cache.Store that returns list of v1.Pod instead of interface{}.
-type PodStore struct {
+// ObjectStore is a convenient wrapper around cache.Store.
+type ObjectStore struct {
 	cache.Store
 	stopCh    chan struct{}
 	Reflector *cache.Reflector
 }
 
-// NewPodStore creates PodStore based on given namespace, label selector and field selector.
+// newObjectStore creates ObjectStore based on given object  selector.
```

Comment:
NIT: double space between object and selector

Reply:
Will fix in subsequent PR as this apparently already merged :)